java-security/src/test/java/com/sap/cloud/security/token/validation/validators/JwtIssuerValidatorTest.java

```diff
@@ -107,11 +107,38 @@ void validationFails_iasIssuerUrl(String iasIssuer) {
 		assertThat(validationResult.getErrorDescription(), startsWith("Issuer is not trusted because issuer "));
 	}
 
+	@ParameterizedTest
+	@CsvSource({ "https://subdomain.accounts400.ondemand.com#anyFragment_keys",
```

Comment:
thanks for adding test cases for xsuaa, but for xsuaa this JwtIssuerValdiator is never called - as of now.

Reply:
-> I've deleted both xsuaa test cases again